src/components/Footer.js

```diff
@@ -1,27 +1,176 @@
+import { useState } from 'react';
 import { Typography } from "@mui/material";
 import Card from "@mui/material/Card";
 import CardContent from "@mui/material/CardContent";
 import Link from "@mui/material/Link";
+import Box from "@mui/material/Box";
+import Stack from "@mui/material/Stack";
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import MailchimpFormContainer from "./MailchimpFormContainer";
 
-export default function Escrow(props) {
-  return (
-    <Card variant="outlined" sx={{mt: 1}}>
-      <CardContent>
-          <CardTextBlock 
-            value={
-              <Link href="https://github.com/humanprotocol/hmt-escrow" target="_blank" rel="noreferrer">
-                HMT Escrow Source Code
-              </Link> }
-           />
-      </CardContent>
-    </Card>
-  );
-}
+import gitImg from '../assets/images/git.png';
+import discordImg from '../assets/images/discord.png';
+import inImg from '../assets/images/in.png';
+import twitterImg from '../assets/images/twitter.png';
+
+export default function Footer(props) {
 
-function CardTextBlock({ value }) {
   return (
-    <Typography variant="body2" textAlign="center">
-      {value}
-    </Typography>
+    <>
+      <Box sx={{paddingTop: '100px', paddingBottom: '100px'}} className='paddingX'>
+        <Stack
+            direction={{ lg: "row", md: "row", sm: "column", xs: "column" }}
+            spacing={2}
+            justifyContent="space-between"
+            sx={{
+              width: "100%",
+            }}
+          >
+            <Box>
+              <Stack
+                direction={{ lg: "row", md: "row", sm: "row", xs: "column" }}
+                spacing={2}
+                sx={{
+                  width: "100%",
+                }}
+              >
+                <MailchimpFormContainer />
```

Comment:
We are not using mailchimp for that. Let's do it in different PR and remove the logic here